src/ios/NewRelicCordovaPlugin.m

```diff
@@ -1,6 +1,9 @@
 //  New Relic for Mobile -- iOS edition
 //
-//  Copyright (c) 2017 - Present New Relic. All rights reserved.
+//  See:
+//    https://docs.newrelic.com/docs/releases/ios for release notes
+//
+//  Copyright (c) 2017 New Relic. All rights reserved.
 //  See https://docs.newrelic.com/docs/licenses/ios-agent-licenses for license details
```

Comment:
We would like to remove the release notes from the comments, can you keep my updates?